### PR TITLE
Adds fucking `article` tag to main content

### PR DESCRIPTION
**Pull request by RubberEyes**
A fucking website needs to be fucking accessible and we need to make sure the stupid machines know what is important in a fucking document so they can read it out to people

### CODE REVIEW

**Comment by LeoColomb** [comment]
Thanks for opening this PR @RubberEyes 👍
Let's use more friendly wording in direct discussions to make sure it does not actually hurt anyone.

While adding this tag usually makes sense, I'm not so sure here:
* [`article`](https://developer.mozilla.org/docs/Web/HTML/Element/article) is to highlight the boundaries of one structured piece of information inside the main content of the page.
* As there is no `main` on this page, adding first a [`main`](https://developer.mozilla.org/docs/Web/HTML/Element/main) would make more sense.
* But then `main` is only useful if there are other elements not part of the main content (mostly `header` and `footer`), which is not the case here.

In short, the [`body`](https://developer.mozilla.org/docs/Web/HTML/Element/body) seems more than enough to tell any client the content is in there.
I'm not sure to see any issues for accessibility so far.

**Comment by RubberEyes** [comment]
A common feature in web browsers is the "reader mode." It just prominently displays the page content without the unnecessary things that websites usually come with, not a problem in this case, but it also enables accessibility functionalities like narrator, font size adjustments, color, etc. Implementation varies between browsers, but some like Firefox look for a `<article>` tag to enable the reader mode.

**Comment by LeoColomb** [comment]
Thanks for your reply @RubberEyes.
I'm afraid that things are a bit more complicated 😅
As you were mentioning Firefox, let's explore Firefox implementation: https://github.com/mozilla/readability

* Readability score is [calculated on based on a couple of aspects](https://github.com/mozilla/readability/blob/e806508e9c1e2007bebe75bff47b53dcaca80903/Readability-readerable.js#L84-L115), more than the presence of `article` tag
  * Applied to the page https://perfectmotherfuckingwebsite.com/, the score is actually pretty good: 46 for a threshold of 20.
  * In the current conditions, the page is perfectly available for the reader mode.
* The Firefox Reader mode in production is based on Readability, with some tweaks.
  * In our case, one specific condition is fired and seems to block its availability: https://searchfox.org/mozilla-central/source/toolkit/components/reader/Readerable.js#80-82
  * In short, the root URI of all domain names is excluded from the reader mode.

It is actually fairly easy to assess the above interpretation: just adding the HTML file path in the URI makes it available to the reader mode https://perfectmotherfuckingwebsite.com/index.html

Based on this analysis, I'm going to close this pull request.
Thanks for your PR anyway, it was quite interesting to dig into the implementation to understand the behavior of the reader mode.
